common/postgres-init.sql

```diff
@@ -0,0 +1,3 @@
+CREATE ROLE cbas_user WITH LOGIN ENCRYPTED PASSWORD 'cbas_password';
+CREATE DATABASE cbas_db OWNER cbas_user;
+\c cbas_db;
```

Comment:
We this line because it was necessary on some systems to install `pgcrypto` in the correct database. If you're not using `pgcrypto` (and I'm not sure catalog should be, but the project we copied did...) then you shouldn't need the `\c cbas_db;` line.

Reply:
As for why you might want `pgcrypto`, catalog uses it to generate UUIDs in SQL. The relevant code is 

https://github.com/DataBiosphere/terra-data-catalog/blob/main/service/src/main/resources/db/changelog.xml#L4
https://github.com/DataBiosphere/terra-data-catalog/blob/main/service/src/main/resources/db/changesets/20220301_datasettable.yaml#L12

Reply:
I suspect we will generate all IDs in the java service, so will remove this for now. Good clue if we need it later though!